src/main/java/com/amazon/jenkins/ec2fleet/fleet/Fleet.java

```diff
@@ -0,0 +1,34 @@
+package com.amazon.jenkins.ec2fleet.fleet;
+
+import com.amazon.jenkins.ec2fleet.FleetStateStats;
+import hudson.util.ListBoxModel;
+
+import java.util.Collection;
+import java.util.Map;
+
+/**
+ * Hide details of access to EC2 Fleet depending on implementation like EC2 Fleet based on EC2 Spot Fleet
+ * or Auto Scaling Group.
+ *
+ * @see SpotFleet
```

Comment:
Add a comment for EC2Fleet and update comment `Hide details of access to EC2 Fleet depending on implementation like EC2 Fleet or Spot Fleet or Auto Scaling Group.

Reply:
Comments added